cmd/kubeadm/app/constants/constants_test.go

```diff
@@ -159,20 +159,10 @@ func TestEtcdSupportedVersion(t *testing.T) {
 			expectedVersion:   nil,
 			expectedError:     errors.New("Unsupported or unknown Kubernetes version(1.99.0)"),
 		},
-		{
-			kubernetesVersion: "1.10.2",
-			expectedVersion:   version.MustParseSemantic("3.1.12"),
-			expectedError:     nil,
-		},
-		{
-			kubernetesVersion: "1.11.0",
-			expectedVersion:   version.MustParseSemantic("3.2.18"),
-			expectedError:     nil,
-		},
```

Comment:
we can technically this etcd test better.
possibly in another PR if you'd prefer.

what can be done is to add test cases by iterating the `SupportedEtcdVersion` map for valid cases and using @bart0sh 's `With...` methods. just an idea.

this way we don't have to bump this test on each minor release.

Reply:
I'll do that in another pr, probably in the next cycle